air/src/constraints/chiplets/bitwise/mod.rs

```diff
@@ -424,10 +424,10 @@ pub const BITWISE_K1_MASK: [Felt; OP_CYCLE_LEN] = [ONE, ONE, ONE, ONE, ONE, ONE,
 
 /// Returns the values from the bitwise periodic columns for the specified cycle row.
 #[cfg(test)]
-fn get_periodic_values(cycle_row: usize) -> [Felt; 2] {
-    match cycle_row {
-        0 => [ONE, ONE],
-        8 => [ZERO, ZERO],
+fn get_periodic_values(cycle_row: crate::RowIndex) -> [Felt; 2] {
```

Comment:
If I `use crate::RowIndex;` then linter complains of unused import as it is only used in tests.

Reply:
Ah, this should be moved in the `air/src/constraints/chiplets/bitwise/tests.rs` file